File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/List.java
package work.socialhub.kmastodon.entity;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * @author uakihir0
 */
public class List implements Serializable {

    @SerializedName("id")
    private String id;
    @SerializedName("title")
    private String title;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: core/src/commonMain/kotlin/work/socialhub/kmastodon/entity/List.kt
package work.socialhub.kmastodon.entity

import com.google.gson.annotations.SerializedName

/**
 * @author uakihir0
 */
class List : java.io.Serializable {
    @SerializedName("id")
    var id: String? = null

    @SerializedName("title")
    var title: String? = null
}
